examples/sample-app/README.md

```diff
@@ -70,8 +85,8 @@ Application Build, Deploy, and Update Flow
 
 This section covers how to perform all the steps of building, deploying, and updating an application on the OpenShift platform.
 
-NOTE: All commands assume the `osc` binary/symlink is in your path and
-the present working directory is the same directory as this README.
+**NOTE**: All commands assume the `osc` binary/symlink is in your path (**VAGRANT USERS**: the `osc` binary/symlink will be located in `/origin/_output/local/go/bin/openshift`).  Also, all commands assume the present working directory is the same directory as this README.
```

Comment:
`/origin/_output/local/go/bin/openshift` suggests absolute path, which is not true, I'd rather see this as `origin/_output/local/go/bin/openshift` or full path with `/data/src/github.com/openshift/origin/_output/local/go/bin/openshift`. Where I prefer the first since it's more compact, the later is too noisy IMHO.